sdk/eventhubs/azure-eventhubs/src/samples/java/com/azure/messaging/eventhubs/EventHubProducerJavaDocCodeSamples.java

```diff
@@ -72,4 +73,42 @@ public void sendEventsFluxSendOptions() {
             () -> System.out.println("Send complete."));
         // END: com.azure.messaging.eventhubs.eventhubproducer.send#publisher-sendOptions
     }
+
+    /**
+     * Code snippet demonstrating how to create an {@link EventDataBatch} and send it.
+     */
+    public void sendEventDataBatch() {
+        final EventHubProducer producer = client.createProducer();
+
+        // BEGIN: com.azure.messaging.eventhubs.eventhubproducer.send#eventdatabatch
+        final Flux<EventData> telemetryEvents = Flux.just(
+            new EventData("92".getBytes(UTF_8)).addProperty("telemetry", "latency"),
+            new EventData("98".getBytes(UTF_8)).addProperty("telemetry", "cpu-temperature"),
+            new EventData("120".getBytes(UTF_8)).addProperty("telemetry", "fps")
```

Comment:
Should we have a ctor overload for String?

Reply:
JS had an overload that took a string, but we removed that because people usually use complex data.. it was easier to create bytes this way.